src/main/java/org/springframework/data/elasticsearch/annotations/Field.java

```diff
@@ -148,4 +149,9 @@
 	 * @since 4.0
 	 */
 	double scalingFactor() default 1;
+
+	/**
+	 * @since 4.0
+	 */
+	int maxShingleSize() default 3;
```

Comment:
```suggestion
	int maxShingleSize() default -1;
```
we should use -1 here to denote a value that is not set (I forgot that we can't use null)